Dockerfile

```diff
@@ -5,7 +5,7 @@ ARG DEBIAN_TAG=12.1-slim
 
 # Stage 1: build artifact and download deps
 
-FROM golang:1.20.7-alpine AS builder
+FROM golang:1.21-alpine AS builder
```

Comment:
not avail yet

Reply:
![image](https://github.com/runatlantis/atlantis/assets/1580956/373017e8-c1ad-4011-9ef6-a2dc44770d05)